pkgs/dart_model/lib/src/dart_model.g.dart

```diff
@@ -514,6 +514,33 @@ extension type StaticTypeDesc.fromJson(Map<String, Object?> node)
   }
 }
 
+extension IterableOfStaticTypeDescExtension on Iterable<StaticTypeDesc> {
+  Iterable<DynamicTypeDesc> whereTypeDynamicTypeDesc() =>
```

Comment:
I worry a bit about exposing these Iterables, especially with the likely high cost of iteration (and map lookups involved) due to the underlying representation here, combined with the lack of composability of them.

I also don't love the naming scheme haha. Maybe just remove the first `Type`? So, `whereDynamicTypeDesc`, etc.